PluginsAndFeatures/azure-toolkit-for-intellij/gradle.properties

```diff
@@ -5,7 +5,7 @@ org.gradle.jvmargs='-Duser.language=en'
 sources=false
 
 intellij_version=IC-221.5080-EAP-CANDIDATE-SNAPSHOT
-rider_version=RD-2022.1-EAP9-SNAPSHOT
+rider_version=RD-2022.1-EAP10-SNAPSHOT
```

Comment:
I guess we would need to bump the version to the release SDK when it is published.